gear-inventory.controller.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('wta.gear-inventory')
+    .controller('GearInventoryController', GearInventoryController);
+
+  GearInventoryController.$inject = ['logger', 'WtaApi'];

```

Comment:
You need to add 'ProfileAccess' and '$state' to the GearInventoryController.$inject

Reference: https://github.com/SEGR-SEATTLEU/FALL-2016/blob/development/src/client/app/create-request/create-request.controller.js